docs/core_systems/service_authorization.md

```diff
@@ -0,0 +1,11 @@
+# Service Authorization
+
+This core system provides the data storage functionality for the information related to the currently and actively offered services within the Local Cloud. It also stores information about the systems that offer and/or can use the previously mentioned services, and optionally data about the devices on which those systems are running.
```

Comment:
This paragraph is about Service Registry, not Authorization. Should change to TODO, if the content is not available yet.